commit d29e72cfc88148b6a22183b7c03a89a32b6f6bbe
Author: Volker Krause <vkrause@kde.org>
Date:   Thu Jul 6 17:17:28 2023 +0200

    Set ndkVersion in build.gradle
    
    Without this binary stripping wont work from Qt 5.15.10 onwards, massively
    increasing the APK size.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4d21668d..466cb858 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -45,7 +45,7 @@ endif()
 
 set(QT_MIN_VERSION 5.15.2)
 if (ANDROID)
-    set(QT_MIN_VERSION 5.15.8) # due to 5.15.8 changing the Gradle version in a non-backward compat way
+    set(QT_MIN_VERSION 5.15.10) # due to 5.15.10 changing NDK versioning
 endif()
 if (QT_MAJOR_VERSION EQUAL "6")
     set(KF_MAJOR_VERSION "6")
diff --git a/src/app/android/build.gradle b/src/app/android/build.gradle
index 8fbc293c..fff5cf52 100644
--- a/src/app/android/build.gradle
+++ b/src/app/android/build.gradle
@@ -48,6 +48,7 @@ android {
     compileSdkVersion androidCompileSdkVersion.toInteger()
 
     buildToolsVersion androidBuildToolsVersion
+    ndkVersion androidNdkVersion
 
     sourceSets {
         main {
diff --git a/src/app/android/gradle.properties b/src/app/android/gradle.properties
index fc750e88..6e98716f 100644
--- a/src/app/android/gradle.properties
+++ b/src/app/android/gradle.properties
@@ -2,6 +2,6 @@
 # SPDX-License-Identifier: BSD-3-Clause
 android.useAndroidX=true
 
-# Gradlde properties set by Qt
+# Gradle properties set by Qt
 org.gradle.jvmargs=-Xmx2500m -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 org.gradle.parallel=true
